src/frame-writer.cpp

```diff
@@ -601,7 +602,9 @@ void FrameWriter::finish_frame(AVCodecContext *enc_ctx, AVPacket& pkt)
         pending_mutex.unlock();
     }
 
-    av_interleaved_write_frame(fmtCtx, &pkt);
+    if (!!av_interleaved_write_frame(fmtCtx, &pkt)) {
```

Comment:
what is the point of double negation? If you want something explicit, write `(... != 0)`

Reply:
That is a habit I gained from reading kernel sources, I'll remove or change it.